src/sections/dataset/dataset-files/files-table/file-info/file-info-cell/file-info-data/FileDate.tsx

```diff
@@ -1,15 +1,17 @@
-import { FileDate as FileDateModel } from '../../../../../../../files/domain/models/FileMetadata'
 import { useTranslation } from 'react-i18next'
+import { FileDate as FileDateModel } from '../../../../../../../files/domain/models/FileMetadata'
 import { DateHelper } from '../../../../../../../shared/helpers/DateHelper'
 
-// TODO: use time tag with dateTime attr https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time
-
 export function FileDate({ date }: { date: FileDateModel }) {
   const { t } = useTranslation('files')
+
   return (
     <div>
       <span>
-        {t(`table.date.${date.type}`)} {DateHelper.toDisplayFormat(date.date)}
+        {t(`table.date.${date.type}`)}{' '}
+        <time dateTime={DateHelper.toDisplayFormat(date.date)}>
```

Comment:
datetime value is the machine-readable value of the datetime, so in there you shouldn't use the `displayFormat` which is a a more human readable but the `toISO8601Format` one.